bindings/ruby/test/test_plugin_ModelTask.rb

```diff
@@ -0,0 +1,1331 @@
+                task.exported_links = [Types.rock_gazebo.LinkExport.new(
+                    port_name: 'test', source_link: 'nested::l', target_link: 'root',
+                    port_period: Time.at(0)
+                )]
+                task.configure
+                task.start
+
+                link_pose = assert_has_one_new_sample(task.test.reader)
+                assert Eigen::Vector3.new(3, 2, 1).approx?(link_pose.position)
+                assert Eigen::Quaternion.from_angle_axis(-0.2, Eigen::Vector3.UnitZ)
+                                        .approx?(link_pose.orientation)
+            end
+
+            it 'the pose\'s update period is controlled by the '\
+                'port_period parameter' do
+                task.exported_links = [Types.rock_gazebo.LinkExport.new(
+                    port_name: 'test', source_link: 'l', target_link: 'root',
+                    port_period: Time.at(0.1)
+                )]
+                task.configure
+                task.start
+                reader = task.test.reader
+                first_pose = assert_has_one_new_sample(reader)
+                second_pose = assert_has_one_new_sample(reader)
+                assert_in_delta(second_pose.time - first_pose.time, 0.1, 0.025)
+            end
+
+            it 'refuses to configure if the source link does not exist' do
+                task.exported_links = [Types.rock_gazebo.LinkExport.new(
+                    port_name: 'test',
+                    source_link: 'does_not_exist', target_link: 'root',
+                    port_period: Time.at(0)
+                )]
+                assert_raises(Orocos::StateTransitionFailed) do
+                    task.configure
+                end
+            end
+
+            it 'refuses to configure if the target link does not exist' do
+                task.exported_links = [Types.rock_gazebo.LinkExport.new(
+                    port_name: 'test',
+                    source_link: 'l', target_link: 'does_not_exist',
+                    port_period: Time.at(0)
+                )]
+                assert_raises(Orocos::StateTransitionFailed) do
+                    task.configure
+                end
+            end
+
+            it 'refuses to configure if the port is already in use' do
+                task.exported_links = [
+                    Types.rock_gazebo.LinkExport.new(
+                        port_name: 'test', source_link: 'l', target_link: 'root',
+                        port_period: Time.at(0)
+                    ),
+                    Types.rock_gazebo.LinkExport.new(
+                        port_name: 'test', source_link: 'l', target_link: 'root',
+                        port_period: Time.at(0)
+                    )
+                ]
+                assert_raises(Orocos::StateTransitionFailed) do
+                    task.configure
+                end
+            end
+        end
+    end
+end
+
```

Comment:
```suggestion
end

```
Missing closing `end` keyword for:

```ruby
describe 'The Rock/Gazebo plugin' do
    include Orocos::Test::Component
    include Helpers

    (...)

end

```